src/web/core/src/test/java/org/geoserver/web/data/resource/ResourceConfigurationPageTest.java

```diff
@@ -120,6 +120,22 @@ public void testBasic() {
         tester.assertComponent(
                 "publishedinfo:tabs:panel:theList:0:content", BasicResourceConfig.class);
     }
+    
+    @Test
+    public void testResourceConfigurationPageStoreNameLabel() {
+    	LayerInfo layer =
+                getGeoServerApplication()
+                        .getCatalog()
+                        .getLayerByName(getLayerId(MockData.BASIC_POLYGONS));
+        login();
+        tester.startPage(new ResourceConfigurationPage(layer, false));
+        
+        tester.assertLabel(
+        		"publishedinfo:tabs:panel:theList:0:content:storeName", 
+        		layer.getResource().getStore().getName());
+        assertNotNull(tester.getComponentFromLastRenderedPage(
```

Comment:
It would probably be best check to check if the label NULL before checking its content.

Reply:
The two assertions have been switched.